dev/cli_md_template.md

```diff
@@ -2,7 +2,13 @@
 
 _WARNING: you are on the master branch, please refer to the docs on the branch that matches your `cortex version`_
 
-## Installing a specific version of the CLI
+## Install CLI
+
+```bash
+pip install cortex
+```
+
+## Install CLI without Python dependency
```

Comment:
Maybe rename this to `Install CLI without Python Client`.

I think with `Install CLI without Python dependency` we may run the risk of the user thinking of `pip install cortex --no-deps`, and so they'll just jump on using that instead.